OpenLaTeXStudio/EditorModule/src/latexstudio/editor/files/FileService.java

```diff
@@ -62,4 +64,25 @@ public static String readFromFile(String filename) {
         
         return content;
     }
+    
+    public static String readFromStream(InputStream stream){

```

Comment:
IOUtils from Apache Commons implements reading input streams to a file. We could use that instead.